garage/np/algos/cem.py

```diff
@@ -113,5 +121,105 @@ def train_once(self, itr, paths):
         logger.log(tabular)
         return rtn
 
+    def process_samples(self, itr, paths):
```

Comment:
is all of this processing actually required for CEM? which of these data does it actually use?